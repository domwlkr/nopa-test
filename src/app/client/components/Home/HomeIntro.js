import React from 'react';
import { Button } from '../index';
import logo from '../../../static/images/Logo_Nopa.svg';
import * as Paths from '../../constants/paths';

const HomeIntro = (props) => {
  return (
    <div className="main-content home-intro">
      <div className="logo">
        <img className="Nopa" alt="Logo" src={logo} />
      </div>

      <h1>Your finances, in one place</h1>
      <p>Track all of your payments by connecting as many bank accounts as you'd like to your Nopa account and get updates on your balance instantly.</p>

      <Button to={Paths.CHOOSE_BANK} className="button">Get started</Button>
    </div>
  );
};

export default HomeIntro;