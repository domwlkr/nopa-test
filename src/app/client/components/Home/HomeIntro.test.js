import React from 'react';
import HomeIntro from './HomeIntro';
import renderer from 'react-test-renderer';

test('Renders correctly', () => {
  const homeIntro = renderer.create(
    <HomeIntro />
  ).toJSON();

  expect(homeIntro).toMatchSnapshot();
});