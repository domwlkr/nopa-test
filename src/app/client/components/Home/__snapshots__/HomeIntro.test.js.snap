// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<div
  className="main-content home-intro"
>
  <div
    className="logo"
  >
    <img
      alt="Logo"
      className="Nopa"
      src="test-file-stub"
    />
  </div>
  <h1>
    Your finances, in one place
  </h1>
  <p>
    Track all of your payments by connecting as many bank accounts as you'd like to your Nopa account and get updates on your balance instantly.
  </p>
  <a
    className="button"
    onClick={[Function]}
    style={Object {}}
  >
    Get started
  </a>
</div>
`;
